app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :order_desc, ->{order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
và user_id thì khác gì nhau?

Reply:
@mabugay1234

Reply:
`validates :user_id` sẽ kiểm tra trường user_id của người dùng có tồn tại hay không, có khoảng trắng không có nil không
`validates :user` kiểm tra xem user có tồn tại không